disperser/apiserver/server.go

```diff
@@ -702,7 +704,7 @@ func (s *DispersalServer) RetrieveBlob(ctx context.Context, req *pb.RetrieveBlob
 			return nil, api.NewResourceExhaustedError(errorString)
 		}
 	}
-
+	s.logger.Debug("checked retrieval blob rate limiting", "requesterID", fmt.Sprintf("%s:%s", origin, RetrievalBlobRateType.Plug()), "duration (ms)", time.Since(stageTimer).Milliseconds())
```

Comment:
nit: could we name labels camelcase? or `time.Since(stageTimer).String()` should format it in a human readable way then we probably don't need to indicate the unit in the label

Reply:
The mixed cap is pretty compatible with golang. Sounds good to let `String()` to pretty print.

Reply:
It's compatible in Golang, but doesn't work nicely when filtering by labels in Loki

Reply:
feel free to start a conversation on this topic, and if it's indeed a good idea, make a migration cross the board. It's not that useful to start ad hoc style at random places.